services/server/src/lib/tutor.ts

```diff
@@ -41,8 +41,11 @@ export async function constructTutorsCache(): Promise<TutorsCache> {
     return {
       id: tutor.id,
       name: tutor.name,
+      email: tutor.email,
       image: tutor.image,
       video: tutor.video,
+      phoneNumber: tutor.phoneNumber,
+      city: tutor.city,
```

Comment:
These are private/personal information and not inteneded to go to the cache. Please revert this change. 

To access this information: 

```ts
// Access persona/private information from `user` and `meta`
// Refersh private information by calling `refetch`.
const {user, meta, refetch} = useCurrentUser()
```